src/modules/fancyzones/editor/FancyZonesEditor/Models/Settings.cs

```diff
@@ -414,35 +418,32 @@ private void ParseDeviceInfoData()
             }
         }
 
-        private void ParseCommandLineArgs()
+        private void ParseCommandLineArgs(int monitor_shift)
         {
             _workArea = SystemParameters.WorkArea;
             Monitor = 0;
 
             string[] args = Environment.GetCommandLineArgs();
-            if (args.Length == 7)
+            if (uint.TryParse(args[(int)CmdArgs.MonitorHandle + (monitor_shift * 6)], out uint monitor))
             {
-                if (uint.TryParse(args[(int)CmdArgs.MonitorHandle], out uint monitor))
-                {
-                    Monitor = monitor;
-                }
+                Monitor = monitor;
+            }
 
-                var parsedLocation = args[(int)CmdArgs.X_Y_Width_Height].Split('_');
-                var x = int.Parse(parsedLocation[0]);
-                var y = int.Parse(parsedLocation[1]);
-                var width = int.Parse(parsedLocation[2]);
-                var height = int.Parse(parsedLocation[3]);
+            var parsedLocation = args[(int)CmdArgs.X_Y_Width_Height + (monitor_shift * 6)].Split('_');

```

Comment:
is there a reason to why the extra spaces where removed? (while creating a PR to master or interns/dev there will be a lot of diffs that might confuse reviewers)

Reply:
Oops, forgot to `ctrl + k` and `ctrl + d`! I'll fix that once we merge with Alex's branch (: